Readme.md

```diff
@@ -61,7 +61,7 @@ Refresh Token.
 
 * A Gmail Account with access <br/> https://support.google.com/mail/answer/56256?hl=en
 
-* New project with `Gmail API` enabled on the API Console. If you want to use **`Listener`**, then enable `Cloud Pub/Sub API` too.
+* New project with `Gmail API` enabled on the API Console. If you want to use **`Listener`**, then enable `Cloud Pub/Sub API` or you can setup a service account with pubsub admin role and use it in configurations.
```

Comment:
Shall we give a reference to the documentation to create a service account?

Reply:
yes. Already added in line 84